lib/piezo.js

```diff
@@ -401,16 +401,37 @@ Piezo.ToSong = function(stringSong, beats) {
   return song;
 };
 
+/**
+ * Play a note for a duration.
+ * @param {string} note - see Piezo.Notes.  Case-insensitive.
+ *   If an octave is not given, the configured default octave
```

Comment:
Perhaps extend this comment to explain what `octave` means in this context (e.g. in the string value `"a4"`, `"a"` is the note and `4` is the octave)?